src/modules/kdenlive/configure

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
```

Comment:
I also wonder why this file is needed. I do not know if MLT can still compile with Qt 4 - so some of the checks may be unnecessary.

Reply:
I reused the checks from `src/modules/qt/configure`, in this way I am at least compatible and not limited to the rest of mlt. If Qt4 support is dropped, then it could be removed from the whole project.

Reply:
My question about why it is needed for typewriter. QDomElement is already used in kdenlivetitle_wrapper.cpp. So it would not be required to add a check for Qt. 

I think that maybe this commit includes two unrelated changes:
1) Add typewriter
2) Add configure script and cmake check for Qt detection?

That might be OK. It is just good for the reviewer to know.

Reply:
Yes, agree, might be redundant, but question appears - whether I should be depent on other module which I am not related to? What is one day the whole `src/modules/qt` is gone, or the checks in qt module will change and I lose my checks I trusted on? And if the qt checks are required by more modules, maybe one should move it higher in the checks tree.
And does it harm to have it repeated? I am able to remove it if reviewer ask for.

Reply:
I opened a PR (https://github.com/mltframework/mlt/pull/641) that moves all dependency checks to the top level CMakeLists.txt.

Reply:
I can wait when PR #641 is accepted then update my code accordingly and commit new changes.

Reply:
That PR does not address the configure and makefile build system to which this belongs. In any case, this is the wrong folder/module for this service. It belongs in `qt` since it depends on a) Qt and b) kdenlivetitle.

Reply:
It has been moved to qt module.